tests/test_early_stopping.py

```diff
@@ -204,5 +204,9 @@ def test_early_stopping(self):
             stopper=stopper,
             use_tqdm=False,
         )
-        self.assertEqual(stopper.number_results, len(losses) // stopper.frequency)
-        self.assertEqual(self.stop_epoch, len(losses), msg='Did not stop early like it should have')
+        self.assertEqual(stopper.number_results, (len(losses) + 2 * stopper.frequency) // stopper.frequency)
+        self.assertEqual(
+            self.stop_epoch,
+            (len(losses) + 2 * stopper.frequency),
+            msg='Did not stop early like it should have',
+        )
```

Comment:
@cthoyt @mali-git @mberr Just for awareness. This describes the new behavior of the training loop when using an early stopper.

Reply:
Yes, that's true. I'll change it 👍